docs/bizmodels-dpgs/pathfinding-pilots.md

```diff
@@ -16,16 +16,20 @@ Announced on 2 March, 2021, this pathfinding pilot will be coordinated jointly b
 
 **DHIS2 for COVID-19:** During the COVID-19 pandemic, **Sri Lanka** created a new [DHIS2 Tracker](https://dhis2.org/sri-lanka-covid-surveillance/) (within 2 days after the first suspected case of the novel coronavirus) to aggregate data for purposes of national-level reporting, while still protecting the privacy and security of individuals who data is stored in the system.  In collaboration with the DHIS2 core developers and Sri Lanka’s DHIS2 team (HISP Sri Lanka), this has ended up being implemented and released for global adoption.
 
+[See the DHIS2 Digital Public Goods nomination.](https://digitalpublicgoods.net/registry/dhis2.html).
+
 ### DPG Example #3, SafeYOU - Armenia's first DPG
 [SafeYOU](https://safeyou.space/) is an application and platform designed to help combat violence against women and girls (VAWG) by providing security functions to those seeking help during emergency situations. Safe YOU works with CSOs, individual professionals, international organisations, and state authorities. It builds bridges for women to service providers, knowledge resources, and community support. It collects valuable, anonymous data that can fill the data gap in the fight against VAWG.  _Read more [here](https://digitalpublicgoods.net/blog/a-virtual-safe-space-for-women/)._
 
-### DPG Example #4, Project AEDES - Philippine's first Pathfinding pilot 
+### DPG Example #4, Project AEDES - Philippine's first Pathfinding pilot
 In January 2021, UNICEF Philippines, in cooperation with the UNICEF Office of Global Innovation and the DPGA began a Digital Public Goods Pathfinding Pilot. Using Project AEDES (Advanced Early Dengue Prediction and Exploration Service), a digital public good used to monitor dengue cases, the two are working together to implement it in select pilot cities in the Philippines and other countries with high incidence of dengue and similar data challenges.
 
 The goal of UNICEF Philippine’s pathfinding pilot is two-fold. First, to find ways that DPGs can benefit from and leverage existing technical country capacity to better existing Technology for Development (T4D) efforts that are both relevant to UNICEF programming and supported by UNICEF Philippines and the Government. Second, to establish a mechanism to transfer that capacity and knowledge to other sectors including education and climate change, including a wider range of government line agencies. As part of this pilot, a local DPG landscape analysis will be done to find out which DPGs are currently in use or being developed, and what additional sectors they can serve.  _Read more [here](https://digitalpublicgoods.net/blog/unicef-philippines-announces-its-first-digital-public-good-pathfinding-pilot/)_.
+
+[See the Project AEDES Digital Public Goods nomination.](https://digitalpublicgoods.net/registry/project-aedes.html)
```

Comment:
Added links to the full nominations for a couple of the examples. A lot of companies have been asking for this!

Reply:
Thanks Abby!